client/src/api/api.js

```diff
@@ -134,24 +128,22 @@ export function deleteReview(appId, reviewId) {
     return fromApi(`v1/apps/${appId}/reviews/${reviewId}`, true, deleteOpts)
 }
 
-export function fromApi(url, auth = false, extraOpts) {
-    const headers = getAuthHeaders()
-
+export async function fromApi(url, auth = false, extraOpts) {
```

Comment:
Yup. Just didn't think to refactor old-code, as we probably want to move most of those to V2 soon anyways. 

Can also just keep the client-instance around like for the V2-instance. Will do this as it's a small change, but  I generally don't refactor too much when implementing new features, but I see that this has touched that part of the code so why not.

Reply:
Actually I will keep it as is for now, as it would be cleaner to have one instance with `v1` as the apiVersion. However that would mean that we would need to change all paths in the api-functions. 

So we should probably remove `fromAPI` altogether, and use the apiV1.request directly in the API-functions. To reduce clutter I will open a PR after this is merged.